torch/_inductor/mkldnn_lowerings.py

```diff
@@ -26,7 +26,99 @@
     ExternKernelChoice,
 )
 from .utils import use_aten_gemm_kernels, use_cpp_gemm_template, use_max_autotune
-from .virtualized import ops, V
+from .virtualized import ops, OpsValue, V
+
+
+def create_int8_compensation(
+    W_tensor: torch.Tensor,
+    packed_weight: ir.TensorBox,
+    x_scale: ir.TensorBox,
+    x_zp: ir.TensorBox,
+    w_scale: ir.TensorBox,
+) -> tuple[bool, ir.TensorBox, Optional[ir.TensorBox]]:
+    use_int8_fast_compensation_path = False
+    weight_compens = None
+    x_w_scale = None
+    if all(
+        isinstance(item, ir.TensorBox) and item.get_name() in V.graph.constants
```

Comment:
Are there any tests to cover this case?

Reply:
I think for the dynamic quantization case, the `scale/zp` of act is not constant.